pkg/crc/constants/constants.go

```diff
@@ -64,3 +61,13 @@ func EnsureBaseDirExists() error {
 	}
 	return nil
 }
+
+// Validate the given driver is supported or not
+func ValidateDriver(driver string) error {
+	for _, d := range SupportedVMDrivers {
+		if driver == d {
+			return nil
+		}
+	}
+	return fmt.Errorf("Unsupported driver: %s, use '--vm-driver' option to provide a supported driver %s", driver, SupportedVMDrivers)
```

Comment:
Should we use `fmt.Errorf` or `crc/errors` ?

Reply:
@gbraad done